packages/remix-dev/compiler/plugins/serverAssetsManifestPlugin.ts

```diff
@@ -1,8 +1,8 @@
 import type { Plugin } from "esbuild";
 import jsesc from "jsesc";
+import { assetsManifestVirtualModule } from "@remix-run/config";
```

Comment:
Why did this need to move to remix-config?

Reply:
we don't, it was due to just moving the `config` dir from dev over wholesale 🙈

Reply:
wait, we need `serverBuildVirtualModule` from it - we could just move that over though https://github.com/remix-run/remix/blob/d8180da3ddac5789fd3cd4feb820b28612140bf7/packages/remix-config/config.ts#L401